app/controllers/users_controller.rb

```diff
@@ -25,7 +25,4 @@ def user_params
      params.require(:user).permit(:name, :email, :address, :city, :state, :zip, :password, :password_confirmation)
    end
 
-   def user_update_params
```

Comment:
If I am not mistaken, this controller is for unregistered/new users who only need the `user_params` method to verify their initial registration information.